src/utils/num.ts

```diff
@@ -147,19 +253,38 @@ export function getHexString(value: string) {
 /**
  * Convert string array to hex-string array
  * @returns format: hex-string array
+ * @example
+ * ```typescript
+ * const stringArray: string[] = ["123", "456", "789"];
+ * const result = getHexStringArray(stringArray);
+ * // result = ["0x7b", "0x1c8", "0x315"]
+ * ```
  */
 export function getHexStringArray(value: Array<string>) {
   return value.map((el) => getHexString(el));
 }
 
 /**
  * Convert boolean to "0" or "1"
+ * @example
```

Comment:
Check everywhere that @returns is present.

Reply:
done.
added @returns to every function